Docs for Midea ccm15 new integration

ocalvo opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Docs for Midea ccm15 new integration


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/94824
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4495
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

ocalvo committed:
Add Docs

ocalvo committed:
Merge branch 'next' into MideaCCM15

c0ffeeca7 commented on source/_integrations/ccm15.markdown:
```diff
@@ -0,0 +1,43 @@
+ha_integration_type: integration
+---
+
+The `ccm15` integration allows you to integrate [Midea CCM15](https://mbt.midea.com/hvac-goods/midea-products-category/vrfs/vrf-controller/central-controller-ccm-15) devices in Home Assistant.
```
```suggestion
The CCM15 integration allows you to integrate [Midea CCM15](https://mbt.midea.com/hvac-goods/midea-products-category/vrfs/vrf-controller/central-controller-ccm-15) devices in Home Assistant.
```

c0ffeeca7 committed:
remove code fence

for integrations that support UI config flow, we use the name as shown in the UI.

c0ffeeca7 reviewed (dismissed):
LGTM. Thank you, @ocalvo :+1:

c0ffeeca7 commented on source/_integrations/ccm15.markdown:
```diff
@@ -0,0 +1,43 @@
+
+{% details "Manual configuration steps" %}
+
+1. Browse to your Home Assistant instance.
+1. Go to **{% my integrations title="Settings > Devices & Services" %}**.
+1. In the bottom right corner, select the
+  **{% my config_flow_start icon domain=page.ha_domain %}** button.
+1. From the list, select **CCM15** and follow the instructions on screen.
```
```suggestion
1. Browse to your Home Assistant instance.
2. Go to **{% my integrations title="Settings > Devices & Services" %}**.
3. In the bottom right corner, select the
  **{% my config_flow_start icon domain=page.ha_domain %}** button.
4. From the list, select **CCM15** and follow the instructions on screen.
```

c0ffeeca7 committed:
fix markdown list numbers

frenck commented on source/_integrations/ccm15.markdown:
```diff
@@ -0,0 +1,43 @@
+ha_category:
+  - Climate
+ha_iot_class: Local Polling
+ha_release: 2023.07.0
```
@c0ffeeca7 You forgot to bump the HA release before merging this PR...

frenck commented on source/_integrations/ccm15.markdown:
```diff
@@ -0,0 +1,43 @@
+---
+title: Midea CCM15 Data Converter
```
This title differs from the PR upstream? Which one is correct @ocalvo?

frenck commented on source/_integrations/ccm15.markdown:
```diff
@@ -0,0 +1,43 @@
+- [Configuration](#configuration)
+- [Climate](#climate)
+
+## Configuration
+
+Adding a CCM15 data converter to your Home Assistant instance can be done via the user interface, by using this My Button:
+{% my config_flow_start badge domain=page.ha_domain %}
+
+{% details "Manual configuration steps" %}
+
+1. Browse to your Home Assistant instance.
+2. Go to **{% my integrations title="Settings > Devices & Services" %}**.
+3. In the bottom right corner, select the
+  **{% my config_flow_start icon domain=page.ha_domain %}** button.
+4. From the list, select **CCM15** and follow the instructions on screen.
+
+{% enddetails %}
```
This is not correct, we have an template for this which should have been included.

Please adjust this in a follow up PR @ocalvo.

/CC @c0ffeeca7

c0ffeeca7 replied:
Let me fix this, since I'm the one who merged this.
#30525
